keras/layers/advanced_activations.py

```diff
@@ -36,3 +37,41 @@ def get_output(self, train):
     def get_config(self):
         return {"name":self.__class__.__name__,
         "input_shape":self.input_shape}
+
+
+class TLinear(MaskedLayer):
```

Comment:
Can I suggest using a longer name? Maybe `ThresholdedLinear`? It wasn't immediately obvious what `TLinear` and `TReLU` mean.